src/eventfd.rs

```diff
@@ -11,6 +11,8 @@ use std::{io, mem, result};
 
 use libc::{c_void, dup, eventfd, read, write};
 
+pub use libc::{EFD_CLOEXEC, EFD_NONBLOCK, EFD_SEMAPHORE};
```

Comment:
nit: commit message summary is too long.
https://chris.beams.io/posts/git-commit/#limit-50